### DISCUSSION TITLE
CPU Fans ramping after update

### DISCUSSION THREAD

**Discussion by platinumpies** [Rem0o/FanControl.Releases#537] [Q&A]
Having some weird issues with my CPU curves after updating fan curve.

For some context I've had my CPU curves setup like this in the past with no issues:
min / max temp
55c - 77c
min / max speed
40% - 90%
hysteresis / response time
3c - 5sec

However the issue im having now is that the CPU fan speed ramps up randomly to like 150%+ even while the temps are below 60c. 

Does anyone know why this is happening

![Screenshot 2022-03-15 145233](https://user-images.githubusercontent.com/9022559/158450758-824d6e44-ca0d-4c6b-80fe-f4711921f01c.png)

**Comment by nollynu** [comment]
Yes, the same for me, the fans are ramping all over the place, I've rolled back to another version, and its back to normal now.
